lib/logging/audit/model.js

```diff
@@ -35,6 +35,24 @@ class Audit extends Model {
         };
     }
 
+    static findBy(target, populate, action, limit) {
```

Comment:
Change params to object method `findBy({target,populate})` this is preferred in libraries so API changes won't break project that use it